src/components/pages/HomePage/NavBar/NavBar.scss

```diff
@@ -0,0 +1,256 @@
+	nav {
+		display: none;
+		position: relative;
+		top: 0;
+		min-height: 100svh;
+		/* max-height: 100vh; */
+		background-color: var(--white);
+		/* height: 100%; */
+		cursor: pointer;
+		pointer-events: auto;
+		width: 100%;
+
+		transition: all 0.4s;
+	}
+
+	nav.active {
+		display: block;
+	}
+
+	nav .head {
+		background: var(--white);
+		color: var(--dark);
+	}
+
+	.nav__section {
+		/* background-color: red; */
+		margin: 10px;
+		border-top: 1px solid #cfcece;
+	}
+
+	.nav-list {
+		display: grid;
+		gap: 10px 10px;
+		/* padding-block: 2rem; */
+		/* font-size: 2rem; */
+
+		margin: 0;
+		list-style: none;
+	}
+
+	.nav-list li {
+		padding: 0.8rem 0.3rem 0.7rem 0.3rem;
+		margin-left: 23px;
+		position: relative;
+		border-radius: 8px;
+	}
+
+	.nav__list-title:hover {
+		/* background: #f6f6f6; */
+		background: var(--menu);
+
+		/* padding: 12px 8px; */
+	}
+
+	.nav__list-title a i {
+		/* transition: all 10s;   */
+		-webkit-transition: all 0.5s ease;
+		-moz-transition: all 0.5s ease;
+		transition: all 0.75s ease;
+	}
+
+	.nav__section-title {
+		padding: 20px;
+		font-size: 12px;
+		font-weight: 500;
+		color: #757575;
+		text-transform: uppercase;
+	}
+
+	.nav-list li {
+		list-style: none;
+		/* margin-bottom: 5px; */
+	}
+
+	.nav-list li a {
+		display: flex;
+		align-items: center;
+		gap: 15px;
+
+		font-size: 14px;
+		font-weight: 500;
+		color: var(--dark);
+		text-decoration: none;
+		/* padding: 12px 8px; */
+		border-radius: 8px;
+		transition: all 0.3s;
+	}
+
+	.nav-list > li:hover > a {
+		/* background: antiquewhite; */
+		padding: 0px 4px;
+	}
+
+	.nav__list-text {
+		flex-grow: 1;
+	}
+
+	.nav__list-submenu li {
+		display: none;
+		flex-direction: column;
+		gap: 0.35rem;
+	}
+
+	.show > li {
+		display: flex;
+	}
+
+	.nav__list-submenu li a {
+		/* background: lightgrey; */
+		border-radius: 8px;
+		padding: 10px;
+	}
+
+	.nav__list-submenu li a:hover {
+		background: lightgrey;
+		color: rgba(36, 190, 9, 0.495);
+	}
+
+	/* Main document */
+	main {
+		padding: 30px;
+	}
+
+	/* Arrow things */
+	.arrow {
+		border: solid black;
+		border-width: 0 2px 2px 0;
+		display: inline-block;
+		padding: 3px;
+		transition: all 0.35s;
+	}
+
+	.right {
+		transform: rotate(-45deg);
+		-webkit-transform: rotate(-45deg);
+	}
+
+	.left {
+		transform: rotate(135deg);
+		-webkit-transform: rotate(135deg);
+	}
+
+	.up {
+		transform: rotate(-135deg);
+		-webkit-transform: rotate(-135deg);
+	}
+
+	.down {
+		transform: rotate(45deg);
+		-webkit-transform: rotate(45deg);
+	}
+
+	.nav-toggle {
+		position: absolute;
+		top: 11%;
+		/* right: 300px; */
+		right: -20px;
+		z-index: 100;
+		border-radius: 90%;
+		width: 35px;
+		height: 35px;
+		background-color: var(--white);
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		/* padding-right: 2px; Keep it center */
+		border: 1px solid black;
+		cursor: pointer;
+		pointer-events: auto;
+
+		transition: all 0.5s;
+	}
+
+	nav.closed .nav-toggle {
+		left: 333px;
```

Comment:
change to percent unit

Reply:
switched to using SCSS variable for better comprehension